docs/tutorials/hello.rst

```diff
@@ -1,72 +1,75 @@
-MicroPython that you want to use it, and ``*`` is Python's way to say
-*everything*. So, ``from microbit import *`` means, in English, "I want to be
-able to use everything from the microbit code library".
-
-The second line::
-
-    display.scroll("Hello, World!")
-
-...tells MicroPython to use the display to scroll the string of characters
-"Hello, World!". The ``display`` part of that line is an *object* from the
-``microbit`` module that represents the device's physical display (we say
-"object" instead of "thingy", "whatsit" or "doodah"). We can tell the display
-to do things with a full-stop ``.`` followed by what looks like a command (in
-fact it's something we call a *method*). In this case we're using the
-``scroll`` method. Since ``scroll`` needs to know what characters to scroll
-across the physical display we specify them between double quotes (``"``)
-within parenthesis (``(`` and ``)``). These are called the *arguments*. So,
-``display.scroll("Hello, World!")`` means, in English, "I want you to use the
-display to scroll the text 'Hello, World!'". If a method doesn't need any
-arguments we make this clear by using empty parenthesis like this: ``()``.
-
-Copy the "Hello, World!" code into your editor and flash it onto the device.
-Can you work out how to change the message? Can you make it say hello to you?
-For example, I might make it say "Hello, Nicholas!". Here's a clue, you need to
-change the scroll method's argument.
+...mówi MicroPythonowi, by zaimportował wszystkie rzeczy potrzebne do pracy
+z BBC micro:bit. Wszystko to jest w module ``microbit`` (moduł to biblioteka
+z wcześniej przygotowanym kodem). Poleceniem ``import`` mówisz MicroPythonowi,
+że chcesz użyć danego modułu, a ``*`` to sposób Pythona, na określenie *wszystkiego*.
+Zatem ``from microbit import *`` oznacza "chcę użyć wszystkich dostępnych funkcji
+z biblioteki microbit".
+
+Druga linia::
+
+    display.scroll("Hello, world!")
+    
+...mówi MicroPythonowi, by wyświetlił przesuwający się ciąg znaków "Hello, 
+world!". ``display`` w tym przypadku to *obiekt*
+(ang. object) z modułu ``microbit``, który reprezentuje fizyczny wyświetlacz
+urządzenia (mówimy "obiekt" zamiast "rzecz" lub "to coś").
+By wydać wyświetlaczowi polecenie, używamy jego nazwy po kropce ``.`` - tak
+naprawdę takie polecenia nazywamy *metodami* (ang. method). W tym przypadku
+używamy polecenia ``scroll`` (ang. przewiń). Polecenie ``scroll``
+musi wiedzieć jakie znaki pokazać na wyświetlaczu. Wpisujemy je ujęte
+w cudzysłów (``"``) i zamknięte w nawiasy (``(`` i ``)``). W programowaniu nazywamy
+to *argumentami* (ang. arguments). Zatem ``display.scroll("Hello, world!")``
+oznacza, po angielsku, "chcę użyć wyświetlacza by pokazać przesuwający się
+tekst 'Hello, world!'". Jeśli metoda nie potrzebuje żadnych argumentów,
+musimy to jasno określić używając pustych nawiasów: ``()``.
+
+Skopiuj powyższy kod do swojego edytora i zaprogramuj go (ang. flash) na
```

Comment:
proponuję użyć słowa "wgraj", po "zaprogramuj" jest trochę zbyt wieloznaczne

Reply:
"Wgraj" wydaje się trochę niszowe, może nie być zrozumiałe dla kogoś kto nie programował wcześniej. Ale nie upieram się, mogę zmienić jeśli chcesz.

Reply:
Wydaje mi się, że jeśli już uczymy programować, to powinniśmy też przy okazji uczyć precyzyjnej terminologii, żeby potem przy kontynuowaniu nauki nie trzeba było wszystkiego "oduczać" i uczyć od nowa z poprawnym słownictwem. Słowo "flash" w języku angielskim także jest raczej niszowe, normalnie znaczy zupełnie co innego.

Reply:
Słuszna uwaga. Zaraz poprawię.